public/css/navbar.css

```diff
@@ -20,102 +41,90 @@
 	text-decoration: none;
 }
 
-.logo-container {
-	display: inline-block;
-	padding: 7px 50px;
-}
-
 .navbar-list {
-	float: right;
 	list-style: none;
 	margin: 0;
-	padding: 0 25px;
+	padding: 0;
+	height: 100%;
 }
 
 .navbar-list li {
 	text-transform: uppercase;
-	font-weight: 500;
-	display: inline-block;
-	padding: 15px 22px;
-}
-
-.dropdown:hover {
-	background-color: #50b5dd;
+	display: inline-flex;
+	padding: 0 22px;
+	height: 100%;
+	align-items: center;
 }
 
 .navbar-list a {
-	color: #2b2b2b;
-	font-size: 14px;
+	color: var(--black);
 	transition: color 0.3s ease;
 }
 
-.navbar-list a:hover {
-	color: #737373;
+.navbar-list a:hover,
+.mobile-navbar-list a:hover {
+	color: var(--green);
 }
 
 .dropdown {
 	position: relative;
-	display: inline-block;
+	display: inline-flex;
+	flex-direction: column;
```

Comment:
After hovering over About, the text will align itself to the bottom. One fix that I've found is that adding `vertical-align: top` will prevent that from happening, but there's probably a better flex way of doing it--I haven't been bale to figure out why it happens in the first place from looking at the browser anyways. 

Before:
![image](https://user-images.githubusercontent.com/57082175/129609278-f9c1819a-3ec4-4a17-a61a-6fe2fa878710.png)


After:
![image](https://user-images.githubusercontent.com/57082175/129609262-59328ed3-858b-4211-9274-6730ec75aef4.png)

For some reason, in the before, the box has a jagged edge on the right too, possibly because the About div is a tiny bit bigger.

Reply:
What browser are you using? I can't replicate it on my end. And the jagged edge is odd because the dropdown width is just 100%.

Reply:
Interesting, this definitely seems to be browser specific. I'm using Firefox 91.0 on a 1080p monitor and this happens consistently every time, but I can't seem to get it to happen on Chrome 92.0

Reply:
![image](https://user-images.githubusercontent.com/22756691/131397058-b6d29d75-735a-444f-9def-5304bca729dc.png)
Using Chrome 92.0, seems that the offset issue is still there although a little less obvious. Also, there looks to be a bit of a gap, not sure what's up with that. Taken at 150% zoom.

![image](https://user-images.githubusercontent.com/22756691/131396836-3b293d93-3d7c-4622-a186-b2ccbeb46ce5.png)
Also, can confirm the issues with Firefox 90.0.